packages/openfeature-server-provider/package.json

```diff
@@ -5,14 +5,15 @@
   "type": "module",
   "types": "build/types/index.d.ts",
   "devDependencies": {
-    "@microsoft/api-extractor": "*",
```

Comment:
The intention of this star was to use the same version as the workspace root. I don't think that was completely reliable though. If another package depend on for instance `api-extractor` there is nothing saying yarn couldn't pick their version. I've now added this as a specific constraint.